### PR TITLE
Ag/edit dashboard

### PR DESCRIPTION
**Pull request by agallant121**
Final styling for dashboard

### CODE REVIEW

File: app/views/dashboard/show.html.erb [hunk 1]
```diff
@@ -1,89 +1,91 @@
...
+		<input type="radio" id="enter_location" name="location" value="enter location" class='enter_location' checked="checked">
+		<label for="enter_location" style="display: block;"><br>Enter a location:</label>
 
-	<div class="row">
-		<div class="col s12 m4 l2"></div>
+		<div class="center-align">
+			<div class="row">
+				<div class="col s12 m4 l8"></div>
+				<div class="col s12 m4 l8">
+					<section style="margin-left: 50%; margin-right: -5%">
```
Comment (TravisBorgsmiller) [inline app/views/dashboard/show.html.erb:41]: This alignment looks suitable

File: app/views/dashboard/show.html.erb [hunk 2]
```diff
@@ -1,89 +1,91 @@
 <script>
 	function grabGeolocation() {
-		 fetch("https://www.googleapis.com/geolocation/v1/geolocate?key=<%= ENV['GOOGLE_API_KEY'] %>", {
-			 method: 'POST'
-		 })
-		 .then((response) => response.json())
-		 .then((data) => {
```
Comment (caachz) [inline app/views/dashboard/show.html.erb:7]: If we can remove these changes we can push this to master

**Comment by caachz** [comment]
looks great! lets just change it to keep the javascript the way it was before before committing.

File: app/views/dashboard/show.html.erb [hunk 3]
```diff
@@ -1,89 +1,91 @@
...
-			 document.getElementById('lat').value=data.location.lat
-			 document.getElementById('lng').value=data.location.lng
-			 lat = data.location.lat;
-			 long = data.location.lng;
-		 })}
+		fetch("https://www.googleapis.com/geolocation/v1/geolocate?key=<%= ENV['GOOGLE_API_KEY'] %>", {method: 'POST'}).then((response) => response.json()).then((data) => {
```
Comment (caachz) [inline app/views/dashboard/show.html.erb:3]: has been corrected

**Pull request merged**